Apps/Src/Temperature.c

```diff
@@ -43,16 +43,15 @@ static uint8_t currentChannel = 0;
 #endif
 
 //TODO: tune PID with actual pack and fans, and then change values below to appropiate value
-#define PROPORTION -1234
-#define INTEGRAL 4321
-#define DERIVATIVE -9876
-#define MAX_FAN_SPEED 4000
-#define DIVISOR 25000
+#define PROPORTION 2
+#define INTEGRAL 250
+#define I_ZONE 3000
+#define I_MAX_ACCUM 500000
+#define MAX_FAN_SPEED 8
```

Comment:
There is already a TOPSPEED define in Fans.h. You can either rename that to this or just use that. Having 2 defines for the same purpose is not a good idea.

Reply:
```static const uint32_t``` is a good alternative for local stuff, but these tunable parameters really should be in the header file

also name your ```#define``` more specific, i.e. ```TEMPERATURE_PID_KP``` or something, tbh make em long so we don't get any conflicts and so their purposes are very crystal clear

Reply:
fixed